src/riak_kv_pb_object.erl

```diff
@@ -189,7 +190,9 @@ process(#rpbfetchreq{queuename = QueueName}, #state{client=C} = State) ->
                                 deleted_vclock = pbify_rpbvc(Vclock)},
                 State};
         {ok, RObj} ->
-            EncObj = riak_object:nextgenrepl_encode(repl_v1, RObj),
+            ToCompress =
```

Comment:
If you are not going to change the value other than via configuration, it is much faster to save the value ToCompress in the state in the init function and match on the state in this branch.

Reply:
change made on state